project/HmrcBuild.scala

```diff
@@ -1,24 +1,19 @@
-import sbt._
 import sbt.Keys._
-import uk.gov.hmrc.PublishingSettings._
-import uk.gov.hmrc.SbtAutoBuildPlugin
+import sbt._
 import uk.gov.hmrc.versioning.SbtGitVersioning
 
 object HmrcBuild extends Build {
 
   import uk.gov.hmrc._
-  import DefaultBuildSettings._
-  import uk.gov.hmrc.{SbtBuildInfo, ShellPrompt}
-  import uk.gov.hmrc.PublishingSettings._
 
   val appName = "agent-mtd-identifiers"
 
   lazy val microservice = Project(appName, file("."))
     .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
     .settings(
-      scalaVersion := "2.11.8",
+      scalaVersion := "2.11.7",
```

Comment:
Accidental copy-and-paste from http-verbs as it was the first library project I could think of and there wasn't a template for creating a library. Thanks for spotting, I'll change it back.